app/controllers/AppealsController.scala

```diff
@@ -198,7 +198,7 @@ class AppealsController @Inject()(val appConfig: AppConfig,
         upload.uploadDetails.flatMap { details =>
           upload.uploadFields.map(
             fields => {
-              val uploadAlgorithm = fields("x-amz-algorithm")
+              val uploadAlgorithm = fields("SHA-256")
```

Comment:
This won't work as it will look for SHA-256 as a key in the upload fields. Maybe we can match on the existing value and convert it to SHA-256 and any other values throw an exception and log a PD?

Reply:
i.e.

```
val uploadAlgorithm = fields("x-amz-algorithm") match {
     case <original value> => "SHA-256"
     case _ => throw new Exception("msg") // have a specific exception for it?
}
```